src/component/@share/organisms/TabBar.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+import { TabBarButton } from '@/component/@share/molecules';
+
+type TabBarProps = {
+  barState: 'Default' | 'Result';
+  onClick: (state: 'Default' | 'Result') => void;
+};
+
+export const TabBar = ({ barState, onClick }: TabBarProps) => {
+  return (
+    <Container>
+      <TabBarButton
+        text="약속 잡기"
+        isActive={barState === 'Default' && true}
```

Comment:
굳이 이럴 필요 없어보입니다! 어차피 비교 연산의 return 값은 boolean 이니까요 :)

```suggestion
        isActive={barState === 'Default'}
```

Reply:
맞다.. 내가 젤 거슬렸던 문법이 이건데 난 바보야. ㅜㅅㅜ